app/test_split_integer.py

```diff
@@ -2,20 +2,25 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    split = split_integer(value=20, number_of_parts=3)
```

Comment:
Please write code according to the test name. The sum of the `split_integer `should be equal to the value.